src/6_branch.js

```diff
@@ -2164,3 +2164,12 @@ Branch.prototype['setAPIResponseCallback'] = wrap(callback_params.NO_CALLBACK, f
 	this._server.onAPIResponse = callback;
 	done();
 }, /* allowed before init */ true);
+
+/***
+ * @function Branch.referringLink
+ * @param {Boolean} withExtendedJourneysAssist - Boolean indicating whether or not to get ReferringLink for extended Journeys Assist scenario.defaults to false.
+ * Gets the referring link from storage (session, local) wih link expiry applied if provided.
+ */
+ Branch.prototype.referringLink = function(withExtendedJourneysAssist) {
```

Comment:
Is there a simple unit test that could be added for this function?

Reply:
current unit tests for branch js are tightly bound to sauce labs which we are planning to remove and rewrite the whole tests.

Reply:
As a workaround, created a new test file to test referringLink.